src/blocks/author-profile/edit.js

```
@@ -24,159 +18,322 @@ import { useAuthors } from '../../js/hooks';
+		contentOrder ? `content-order-${contentOrder}` : '',
+	]
+		.filter(Boolean)
+		.join(' ');
 
+	const blockProps = useBlockProps({
+		className: blockClassName,
+		style: blockStyles,
+	});
+
+	// Handle author selection
+	const handleSelectAuthor = ([authorId]) => {
```

ESLint

[error] 'authorId' is already declared in the upper scope on line 38 column 3.

(no-shadow)